salt/metalk8s/repo/offline.sls

```diff
@@ -57,8 +77,4 @@ Refresh yum cache:
       - cmd: Refresh yum cache
 
 Repositories configured:
-  test.succeed_without_changes:
-    - require:
-{%- for repository_name in repo.repositories.keys() %}
-      - pkgrepo: Configure {{ repository_name }} repository
-{%- endfor %}
```

Comment:
If you remove this you need to add a `require_in` in `Configure {{ repostiroy_name }} repository` state

Reply:
It's actually not really needed since there is a require_in on it in the yum plugin versionlock package install which requires repositories to be setup... so
repo setup > yum install yum-plugin-version > repositories configured

But I could add it to be more explicit.

Reply:
Even if it's implicit for the moment because of the `require_in` on yum-version like I said in my comment just above (https://github.com/scality/metalk8s/pull/2082/files#r351709055) if we remove the `require_in` on yum-versionlock we want this `Repositories configured` state to require all `Configure ... ` states

Reply:
Yeah, sure, will do it.